extensions/email/src/main/java/org/mule/extension/email/api/EmailContent.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.api;
+
+import org.mule.runtime.extension.api.annotation.Parameter;
+
+/**
+ * Represents and enables the construction of the content
+ *
+ * @since 4.0
+ */
+public class EmailContent
+{
+
+    /**
+     * The text body of the message content.
+     */
+    @Parameter
+    private String body;
```

Comment:
the attribute is optional (see the `send` Operation), but defaults to the payload. if the user wants to explicitly provide an empty content it must write an empty body.